udapi/block/tokenize/onwhitespace.py

```diff
@@ -90,8 +90,8 @@ def process_tree(self, root):
             node.ord = i
 
             if i == 1 and spaces_before:
-                node.misc["SpacesBefore"] = spaces_before.translate(escape_whitespace_table)
+                node.misc["SpacesBefore"] = spaces_before.translate(self.escape_whitespace_table)
             if not spaces_after:
                 node.misc["SpaceAfter"] = 'No'
             elif spaces_after != " ":
```

Comment:
Alternatively, `escape_whitespace_table` could be a module-level constant, instead of class-level, but I have no strong preference here, so I will merge this now.